### PR TITLE
add makegraph2

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
## PR checklist

- Add Makegraph2
- Add manual version reporting for all controlfreec modules. New version 11.6b not reported when running `-version`, but still 11.6

### CODE REVIEW

File: modules/nf-core/controlfreec/makegraph2/meta.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      type: file
+      description: ratio file generated by FREEC
+      pattern: "*.ratio.txt"
+  - baf:
+      type: file
+      description: .BAF file generated by FREEC
+      pattern: "*.BAF"
+  - ploidy:
+      type: integer
+      description: Ploidy value for which graph should be created
```
Comment (FriederikeHanssen) [inline modules/nf-core/controlfreec/makegraph2/meta.yml:35]:
```suggestion
```

**Comment by FriederikeHanssen** [comment]
@nf-core-bot fix linting

**Comment by FriederikeHanssen** [comment]
Unsure why the checksums on the pngs don't match, the images look the same to me: 
test_BAF_conda
test_BAF_docker

**Comment by maxulysse** [comment]
either we merge with conda checksum failings or we remove checksum for the png

**Review by maxulysse — APPROVED** [review]

**Comment by FriederikeHanssen** [comment]
well I completely forgot about this PR and recreated it yesterday 🙈 can we just exclude conda?

**Pull request merged**